### PR TITLE
Add uploading and patching functions

### PR DESCRIPTION
**Pull request by kosaniak**
This change is [Reviewable](https://reviewable.io/reviews/openprocurement/openprocurement.client.python/40)

### CODE REVIEW

File: openprocurement_client/client.py [hunk 1]
```diff
@@ -549,6 +559,33 @@ def upload_award_document(self, file_, tender, award_id):
                      getattr(getattr(tender, 'access', ''), 'token', '')}
         )
 
+    @verify_file
+    def upload_reasons_for_cancellation_of_qualification_document(self, file_, tender, award_id):
```
Comment (leits) [inline openprocurement_client/client.py]: upload_award_document

**Comment by selurvedu** [comment]
Все решта добре. Треба буде ще юніт-тести, але не зараз.

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

_[openprocurement_client/client.py, line 563 at r1](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/40#-KUMJSbu_3dANqsHncFi-r1-563:-KUNTEOFSp193UI1ZST_:b32v57) ([raw file](https://github.com/openprocurement/openprocurement.client.python/blob/62f040295b972fd15508b19b9f072405a1b0d663/openprocurement_client/client.py#L563)):_
<details><summary>

<i>Previously, Leits (Andrew Leitsius) wrote…</i></summary>



> upload_award_document
> </details>
> +1

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/40#-:-KUNU7jowN4U8tMsiWGS:b7ynpe3)_

**Comment by selurvedu** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/40)_

**Comment by leits** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/40)_

**Comment by selurvedu** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

_[openprocurement_client/client.py, line 563 at r1](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/40#-KUMJSbu_3dANqsHncFi-r1-563:-KURO0yBDv8_eLXoFv0X:b32w0x) ([raw file](https://github.com/openprocurement/openprocurement.client.python/blob/62f040295b972fd15508b19b9f072405a1b0d663/openprocurement_client/client.py#L563)):_
<details><summary>

<i>Previously, selurvedu wrote…</i></summary>



> +1
> </details>
> OK

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/40#-:-KURO3z29xvRuiw-tw-C:bnfp4nl)_

**Comment by selurvedu** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

_[openprocurement_client/client.py, line 563 at r1](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/40#-KUMJSbu_3dANqsHncFi-r1-563:-KUROjqtRsfll1dWTbjz:b1iu1dq) ([raw file](https://github.com/openprocurement/openprocurement.client.python/blob/62f040295b972fd15508b19b9f072405a1b0d663/openprocurement_client/client.py#L563)):_

> Merged
> 1 unresolved discussion in status "Blocking"

:expressionless:

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/40)_

**Pull request merged**